> Actually, `configparser` is not used by `cc-plugin-ugrid`. Can you paste the full traceback?


```
2.968 Collecting cc-plugin-ugrid (from -r requirements.txt (line 10))
2.990   Downloading cc-plugin-ugrid-0.1.0.tar.gz (26 kB)
2.999   Installing build dependencies: started
3.662   Installing build dependencies: finished with status 'done'
3.663   Getting requirements to build wheel: started
3.982   Getting requirements to build wheel: finished with status 'error'
3.987   error: subprocess-exited-with-error
3.987
3.987   × Getting requirements to build wheel did not run successfully.
3.987   │ exit code: 1
3.987   ╰─> [33 lines of output]
3.987       /tmp/pip-install-mkv3914p/cc-plugin-ugrid_61f9a94deeea4c1a9b73f3aad6b5df12/versioneer.py:564: SyntaxWarning: invalid escape sequence '\s'
3.987         mo = re.search(r'=\s*"(.*)"', line)
3.987       Traceback (most recent call last):
3.987         File "/usr/local/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 389, in <module>
3.987           main()
3.987           ~~~~^^
3.987         File "/usr/local/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 373, in main
3.987           json_out["return_val"] = hook(**hook_input["kwargs"])
3.987                                    ~~~~^^^^^^^^^^^^^^^^^^^^^^^^
3.987         File "/usr/local/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 143, in get_requires_for_build_wheel
3.987           return hook(config_settings)
3.987         File "/tmp/pip-build-env-19g6jhqi/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 331, in get_requires_for_build_wheel
3.987           return self._get_build_requires(config_settings, requirements=[])
3.987                  ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
3.987         File "/tmp/pip-build-env-19g6jhqi/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 301, in _get_build_requires
3.987           self.run_setup()
3.987           ~~~~~~~~~~~~~~^^
3.987         File "/tmp/pip-build-env-19g6jhqi/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 512, in run_setup
3.987           super().run_setup(setup_script=setup_script)
3.987           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^
3.987         File "/tmp/pip-build-env-19g6jhqi/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 317, in run_setup
3.987           exec(code, locals())
3.987           ~~~~^^^^^^^^^^^^^^^^
3.987         File "<string>", line 19, in <module>
3.987         File "/tmp/pip-install-mkv3914p/cc-plugin-ugrid_61f9a94deeea4c1a9b73f3aad6b5df12/versioneer.py", line 1480, in get_version
3.987           return get_versions()["version"]
3.987                  ~~~~~~~~~~~~^^
3.987         File "/tmp/pip-install-mkv3914p/cc-plugin-ugrid_61f9a94deeea4c1a9b73f3aad6b5df12/versioneer.py", line 1412, in get_versions
3.987           cfg = get_config_from_root(root)
3.987         File "/tmp/pip-install-mkv3914p/cc-plugin-ugrid_61f9a94deeea4c1a9b73f3aad6b5df12/versioneer.py", line 342, in get_config_from_root
3.987           parser = configparser.SafeConfigParser()
3.987                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
3.987       AttributeError: module 'configparser' has no attribute 'SafeConfigParser'. Did you mean: 'RawConfigParser'?
3.987       [end of output]
3.987
3.987   note: This error originates from a subprocess, and is likely not a problem with pip.
3.993 error: subprocess-exited-with-error
3.993
3.993 × Getting requirements to build wheel did not run successfully.
3.993 │ exit code: 1
3.993 ╰─> See above for output.
3.993
3.993 note: This error originates from a subprocess, and is likely not a problem with pip.
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -39,9 +39,6 @@ RUN dnf -y install openssl-devel bzip2-devel libffi-devel sqlite-devel \
     # Clean up source
     cd / && rm -rf /usr/src/Python-3.11.6*
 
-# Install udunits2 and set UDUNITS2_XML_PATH
-RUN dnf -y install udunits2-devel udunits2
-ENV UDUNITS2_XML_PATH=/usr/share/udunits/udunits2.xml
 
 # Startup Shell script
 COPY contrib/docker/my_init.d/run.sh /etc/run.sh

--- a/cchecker_web/processing.py
+++ b/cchecker_web/processing.py
@@ -1,7 +1,5 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-from __future__ import print_function
-from __future__ import unicode_literals
 from compliance_checker.runner import CheckSuite
 import six
 import base64

--- a/requirements.txt
+++ b/requirements.txt
@@ -7,7 +7,7 @@ compliance-checker==5.3.0
 gunicorn>=20.1.0
 cc-plugin-ncei
 cc-plugin-glider
-cc-plugin-ugrid
+cc-plugin-ugrid >=1.1.0
 # we only need this for wheel based apps, anything installing from source or conda-forge is OK with older versions
 # see https://github.com/Unidata/netcdf4-python/issues/1151 for more context
 netcdf4>=1.6.3

--- a/requirements.txt
+++ b/requirements.txt
@@ -7,7 +7,7 @@ compliance-checker==5.3.0
 gunicorn>=20.1.0
 cc-plugin-ncei
 cc-plugin-glider
-cc-plugin-ugrid >=1.1.0
+cc-plugin-ugrid>=1.1.0
 # we only need this for wheel based apps, anything installing from source or conda-forge is OK with older versions
 # see https://github.com/Unidata/netcdf4-python/issues/1151 for more context
 netcdf4>=1.6.3

